docsource/buildandconfigure.rst

```diff
@@ -3,79 +3,30 @@
 Build, configure, and deploy
 ============================
 
-These instructions assume the reader is familiar with the process of deploying
-a KBase module, including the `runtime <https://github.com/kbase/bootstrap>`_
-and `dev_container <https://github.com/kbase/dev_container>`_, and has access to
-a system with the KBase runtime installed. These instructions are based on the
-``kbase-image-v26`` runtime image.
-
-Unlike many modules the WSS can be built and tested outside the
-``dev_container``, but the ``dev_container`` is required to build and test the
-scripts. These instructions are for deploying the server and so do not
-address the scripts. Building outside the ``dev_container`` means the Makefile
-uses several default values for deployment - if you wish to use other values
-deploy from the ``dev_container`` as usual.
-
 Build the workspace service
 ---------------------------
 
-First checkout the ``dev_container``::
-
-    /kb$ sudo git clone https://github.com/kbase/dev_container
-    Cloning into 'dev_container'...
-    remote: Counting objects: 1097, done.
-    remote: Total 1097 (delta 0), reused 0 (delta 0), pack-reused 1097
-    Receiving objects: 100% (1097/1097), 138.81 KiB, done.
-    Resolving deltas: 100% (661/661), done.
-
-.. note::
-   In the v26 image, ``/kb`` is owned by ``root``. As an alternative to
-   repetitive ``sudo`` s, ``chown`` ``/kb`` to the user.
-
-Bootstrap and source the user environment file, which sets up Java and Perl
-paths which the WSS build needs::
-
-    /kb$ cd dev_container/
-    /kb/dev_container$ sudo ./bootstrap /kb/runtime/
-    /kb/dev_container$ source user-env.sh
-
-Now the WSS may be built. If building inside the ``dev_container`` all the
-dependencies from the ``DEPENDENCIES`` file are required, but to build outside
-the ``dev_container``, only the ``jars`` and ``workspace_deluxe`` repos are
-necessary::
+Get the code::
 
     ~$ mkdir kb
     ~$ cd kb
```

Comment:
at some point those lines could be removed since they are not needed

Reply:
Fixed